MIGRATION.md

```diff
@@ -1,5 +1,15 @@
 ## Migration Guide
 
+### Migrating from v3 to v4
+
+V4 has dropped support for use through Bower. If you have been using Bower, you will need to follow the [bower to npm migration guide](https://origami.ft.com/docs/tutorials/bower-to-npm/).
+
+When using Sass, you will need to add `node_modules` to your `includePath` as well as either:
```

Comment:
Should this section also mention the namespace required on the JS import?

Reply:
actually i think I should remove this section, and just point people towards the guide!